.github/CONTRIBUTING.md

```diff
@@ -203,14 +203,15 @@ of the pybind11 repo.
 [`clang-tidy`][clang-tidy] performs deeper static code analyses and is
 more complex to run, compared to `clang-format`, but support for `clang-tidy`
 is built into the pybind11 CMake configuration. To run `clang-tidy`, the
-following recipe should work. Files will be modified in place, so you can
-use git to monitor the changes.
+following recipe should work. Run the `docker` command from the top-level
+directory inside your pybind11 git clone. Files will be modified in place,
+so you can use git to monitor the changes.
 
 ```bash
-docker run --rm -v $PWD:/pybind11 -it silkeh/clang:10
-apt-get update && apt-get install python3-dev python3-pytest
-cmake -S pybind11/ -B build -DCMAKE_CXX_CLANG_TIDY="$(which clang-tidy);-fix"
-cmake --build build
+docker run --rm -v $PWD:/mounted_pybind11 -it silkeh/clang:12
+apt-get update --yes && apt-get install --yes python3-dev python3-pytest
+cmake -S /mounted_pybind11/ -B build -DCMAKE_CXX_CLANG_TIDY="$(which clang-tidy);-fix"
+cmake --build build -- --keep-going -j 2
```

Comment:
I'm not sure why 2 is used here, we don't what the ideal number is. We could use `--parallel=2` instead if you want to make it a bit more obvious that it's the number of build threads (though maybe `-j` is so common that it's best). Also, this needs to be before the `--`, it should go through CMake, not directly to the build backend.

```suggestion
cmake --build build -j 2 -- --keep-going
```

Reply:
> I'm not sure why 2 is used here, we don't what the ideal number is. We could use `--parallel=2` instead if you want to make it a bit more obvious that it's the number of build threads (though maybe `-j` is so common that it's best). Also, this needs to be before the `--`, it should go through CMake, not directly to the build backend.

Applied (by copying the exact command from the workflow file).

In the meantime I stumbled over missing `-D...` options. Added here. I also added a comment in the workflow file, to make it less likely that the documentation is out-of-sync.